gitops/github/README.md

```diff
@@ -0,0 +1,5 @@
+# GitOps CI/CD with GitHub actions
```

Comment:
Are we only supporting Github Actions for the Manifest Generation pipeline? What about Image Tag Release pipeline?

Reply:
can be in another pr. this pr is already quite big. wdyt?